bot/main.py

```diff
@@ -27,6 +28,16 @@
     logging.basicConfig(level=LOGGING_LEVEL)
 
 
+spam_log_file_clear_interval = 3
```

Comment:
I think this interval is too high. I propose 1 second if we only check for "# of messages per interval". Should we also check for message uniqueness then 3 is OK.

Reply:
I tested up to 5 seconds internal and it still feels unnatural to write in more than 3 channels in that amount of time. But made it 2 for now, we can adjust further